packages/react/src/primitives/Menu/MenuButton.tsx

```diff
@@ -23,7 +23,7 @@ export const MenuButton = React.forwardRef<
       isDisabled,
       isLoading,
       size,
-      style,
+      style = {},
```

Comment:
As `style` is used in the dep array for the memoized return value of `useStyles` (see [source](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/src/primitives/shared/styleUtils.ts#L164)), we should avoid setting a default parameter of an empty object here. The empty object will have a new reference on every render, which causes a re-evaluation of the aforementioned `useMemo` in `useStyles`